EXTERNAL throw_exception(str)
=== function throw_exception(str)
EXCEPTION: {str}